samples/openthread/cli/sample.yaml

```diff
@@ -42,3 +42,15 @@ tests:
       - nrf52840dk_nrf52840
       - nrf52840dongle_nrf52840
       - nrf21540dk_nrf52840
+  # Build integration regression protection.
+  #
+  # Use the nrf_security_build_integration tag to ensure this is built on CI
+  # when crypto library changes are made.
+  integration.nrf_security.openthread:
+      tags: nrf_security_build_integration
+      build_only: True
+      extra_args: CONFIG_NRF_SECURITY=y
```

Comment:
Note: We enable mcuboot where possible just to ensure that it builds. It's just an extra check. For this sample, nrf5340_ns has a flash overflow if mcuboot is enabled, so it isn't added here.